### PR TITLE
Apex-NoRecoil | Python Build

### PR DESCRIPTION
**Pull request by krampus-nuggets**
Hi there, I have rewritten your original AHK script in Python. Python version is meant to be used as a CLI with the following being implemented:

* Toggle for enabling/disabling the recoil-control functionality.
* OCR recognition of which weapon is in slots 1 and 2.
* Key bindings are set to the keyboard characters 1 and 2 for updating and applying the recoil patterns based on which weapons are in which weapon slots.
* I've personally tested and confirmed stability and performant usage. Any additional optimization feedback is welcomed.

### CODE REVIEW

File: python/main.py
Comment (mgsweet) [inline python/main.py:28]: coool

**Review by mgsweet — CHANGES_REQUESTED** [review]
1. rename python/ReadMe.md to python/README.md
2. could you help me create a contributor section in the main README.md with your name there.
3. also it would be great if you can add a section in the main README.md to tell those who are not familiar with AHK to take a look at the python version.

**Comment by krampus-nuggets** [comment]
Updates have been applied:

* File has been renamed to README in python dir
* Main repo README content has been updated and Contributor table has been added as well

On a side note, when you get a moment you could implement this tool to automatically generate a contributors table in the main README file - https://allcontributors.org/#

@mgsweet

**Review by mgsweet — APPROVED** [review]
Respect!

**Comment by mgsweet** [comment]
merge it yourself when u think it is ready.

**Comment by mgsweet** [comment]
> https://allcontributors.org/#

OT every day. Will take a look this night

**Comment by krampus-nuggets** [comment]
@mgsweet - I don't have write access, so I can't merge the pull request.

**Comment by mgsweet** [comment]
@krampus-nuggets I think I have invited you. anyway, i would do this

**Comment by mgsweet** [comment]
@all-contributors please add @krampus-nuggets  for code

**Comment by mgsweet** [comment]
@krampus-nuggets Take a look at the latest README. Send me a pull request if you have anything else to add.

**Pull request merged**
